app/controllers/api/v1/visualization/clusters_controller.rb

```diff
@@ -0,0 +1,172 @@
+        before_action :check_study_view_permission
+
+        # shared
+        cluster_param_docs = [{
+            name: :accession,
+            in: :path,
+            description: 'Accession of study',
+            required: true,
+            type: :string
+          },
+          {
+            name: :annotation_name,
+            in: :path,
+            description: 'Name of annotation, as defined in cluster file',
+            type: :string
+          },
+          {
+            name: :annotation_type,
+            in: :query,
+            description: 'Type of annotation.  One of “group” or “numeric”.',
+            type: :string,
+            enum: VALID_TYPE_VALUES
+          },
+          {
+            name: :annotation_scope,
+            in: :query,
+            description: 'Scope of annotation.   One of “study” or “cluster”.',
+            type: :string,
+            enum: VALID_SCOPE_VALUES
+          },
+          {
+            name: :subsample,
+            in: :query,
+            description: 'Number of cells to use as subsample threshold.  Omit parameter if not subsampling (i.e. if threshold is “all cells”).',
+            type: :string
+          },
+          {
+            name: :consensus,
+            in: :query,
+            description: 'Statistical parameter to use for consensus, e.g. “mean”.  Omit parameter if not applying consensus parameter.',
+            type: :string,
+            enum: VALID_CONSENSUS_VALUES
+          }]
+
+        cluster_response_docs = [
+          {code: 200, description: 'Cluster visualization json object'},
+          {code: 401, description: ApiBaseController.unauthorized},
+          {code: 403, description: ApiBaseController.forbidden('view study')},
+          {code: 404, description: ApiBaseController.not_found(Study)},
+          {code: 406, description: ApiBaseController.not_acceptable},
+        ]
+
+        swagger_path '/studies/{accession}/clusters' do
+          operation :get do
+            key :tags, [
+                'Visualization'
+            ]
+            key :summary, 'Get the default cluster and its sub-clusters for a study'
+            key :description, 'Get the default cluster group and its constituent cluster annotations'
+            key :operationId, 'study_clusters_path'
+            cluster_param_docs.each { |doc| parameter(doc) }
+            cluster_response_docs.each { |doc| response doc[:code], doc }
+          end
+        end
+
+        # note that 'index' returns the default cluster, rather than a list of all clusters
+        # see the 'show' method for documentation of the return type
+        def index
+          begin
+            cluster = @study.default_cluster
+            if cluster.nil?
+              render json: {error: 'No default cluster exists'}, status: 422 and return
+            end
+            render json: self.class.get_cluster_viz_data(@study, cluster, params)
+          rescue => e
+            byebug
```

Comment:
I think this is probably a debug leftover - if there are errors we should be reporting them to Sentry and also the user.